sdk/inc/azure/iot/az_iot_pnp_client.h

```diff
@@ -565,42 +566,50 @@ AZ_NODISCARD az_result az_iot_pnp_client_twin_property_builder_end_component(
  *
  * https://docs.microsoft.com/en-us/azure/iot-pnp/concepts-convention#writable-properties
  *
- * The payload will be of the form
+ * The payload will be of the form:
  *
  * **Without Component**
  * @code
- * {
- *   "<property_name>":{
- *     "ac": <ack_code>,
- *     "av": <ack_version>,
- *     "ad": "<ack_description>",
- *     "value": 23
- *   }
- * }
+ * //{
+ * //  "<property_name>":{
+ * //    "ac": <ack_code>,
+ * //    "av": <ack_version>,
+ * //    "ad": "<ack_description>",
+ * //    "value": 23
+ * //  }
+ * //}
  * @endcode
  *
+ * To send a status for a property belonging to a component, first call the
+ * az_iot_pnp_client_twin_property_builder_begin_component() API to prefix the payload with the
+ * necessary identification. The API call flow would look like the following with the listed JSON payload being generated.
+ *
  * **With Component**
  * @code
- * {
- *   "<component_name>": {
- *     "__t": "c",
- *     "<property_name>": {
- *       "ac": <ack_code>,
- *       "av": <ack_version>,
- *       "ad": "<ack_description>",
- *       "value": 23
- *     }
- *   }
- * }
+ * 
+ * az_iot_pnp_client_twin_property_builder_begin_component()
+ * az_iot_pnp_client_twin_begin_property_with_status()
+ * az_iot_pnp_client_twin_end_property_with_status()
```

Comment:
how do you add property value?

Reply:
added a comment where user value would go